tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,9 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
+        self.manifest_url_app = None
```

Comment:
I really don't understand the usage of manifest_url_app. Could somebody please explain it to me?